tools/yaml-templates/ios-test.yml

```diff
@@ -112,15 +112,15 @@ steps:
 
   - task: Bash@3
     displayName: 'iOS UI/E2E Tests'
-    condition: eq(variables['Build.Reason'], 'PullRequest') # Run E2E only for PR builds temporarily until the iOS E2E issue is resolved. 
+    condition: eq(variables['Build.Reason'], 'PullRequest') # Run E2E only for PR builds temporarily until the iOS E2E issue is resolved.
```

Comment:
not a big deal but I'm not sure why these line endings have been changed

Reply:
It seems like an auto-formatter job; it removed the extra space after the full stop.